test/artefacts/marriage-abroad/american-samoa/uk/partner_british/opposite_sex.txt

```diff
@@ -27,8 +27,6 @@ You should also check with the local authorities in American Samoa to find out i
 
 You normally have to pay fees for consular services in the local currency - these are shown in the list of [consular fees for American Samoa](/government/publications/american-samoa-consular-fees).
 
-You can pay by cash or credit card, but not by personal cheque.
-
```

Comment:
I'm not sure I understand how you've decided that changes like these to the regression test artefacts are OK.